### ISSUE TITLE
stanza.grouping produces an error: Cannot read property 'groupBy' of undefined

### ISSUE THREAD

**Issue by dsatoh** [togostanza/ts#59]
I got an error described below by calling `stanza.grouping` in `index.js`.

```
% ts version
ts version 0.0.18 (go1.12 darwin/amd64)
```

## how to reproduce the error

```
% ts new test

% cat test/index.js
Stanza(function(stanza, params) {
  let ary = [
    {x: 1, y: 1, z: 3},
    {x: 1, y: 2, z: 4},
    {x: 2, y: 1, z: 5},
    {x: 2, y: 2, z: 6},
    {x: 1, y: 2, z: 7},
    {x: 2, y: 1, z: 8}
  ];

  console.log(stanza.grouping(ary, ['x', 'y'], 'z'));

  stanza.render({
    template: "stanza.html",
    parameters: {
      greeting: "Hello, world!"
    }
  });
});
```

This stanza will cause,

```
Uncaught TypeError: Cannot read property 'groupBy' of undefined
    at a (stanza.js:97)
    at Object.grouping (stanza.js:104)
    at (index):21
    at stanza.js:156
    at d (index.js:160)
    at v (index.js:207)
    at g (index.js:195)
```

## my solution

Please use this if you are acceptable.

* provider/assets-src/js/stanza.js

```diff
@@ -92,6 +92,7 @@ export default function initialize(descriptor) {
             }
           }, []);

+          const self = this;
           return (function _grouping(rows, keys) {
             const [currentKey, ...remainKeys] = keys;

@@ -103,7 +104,7 @@ export default function initialize(descriptor) {
               return rows.map((row) => fetch(row, currentKey.key));
             }

-            return this.groupBy(rows, (row) => {
+            return self.groupBy(rows, (row) => {
               return fetch(row, currentKey.key);
             }).map(([currentValue, remainValues]) => {
               const nextKey = remainKeys[0];
```

**Comment by ursm** [comment]
Thank you for repoting. The issue should be resolved at [v0.0.19](https://github.com/togostanza/ts/releases/tag/v0.0.19). Could try this?

**Comment by dsatoh** [comment]
@ursm Thank you for new release ts v0.0.19. I got my stanza working properly.
